spec/migrations/20150813202625_set_missing_password_digest_in_authentications_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+require Rails.root.join('db/migrate/20150813202625_set_missing_password_digest_in_authentications')
+
+describe SetMissingPasswordDigestInAuthentications do
+  let(:auth_stub) { migration_stub(:Authentication) }
+
+  before do
+    auth_stub.create!([{:password => 'fred', :password_digest => 'fred'},
+                       {:password => 'barney'}])
+  end
+
+  migration_context :up do
+    it 'with blank password_digest' do
+      migrate
+      auth_stub.all.each do |auth|
+        auth.password_digest.should_not be_nil
```

Comment:
This would be better with real input/output tests.  Put a real encrypted password in the password field, and verify that it actually bcrypts it to a known value.